charts/kube-prometheus-stack/Chart.yaml

```
@@ -17,7 +17,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 25.2.0
+version: 25.1.2
```

New feature should raise minor version.

```suggestion
version: 25.3.0
```

---

truly appreciate